components/parquet_ext/src/prune/equal.rs

```diff
@@ -436,7 +440,7 @@ mod tests {
         // | 1  | 2  | 3  |
         // | 2  | 3  | 4  |
         let row_groups = vec![vec![0, 1, 2], vec![1, 2, 3], vec![2, 3, 4]];
-        let is_equal = |pos: ColumnPosition, val: &ScalarValue, negated: bool| -> bool {
+        let is_equal = |pos: ColumnPosition, val: &ScalarValue, negated: bool| -> Option<bool> {
```

Comment:
Why change to Option here? when will we return None?

Reply:
When the compare result is unknown.